src/main/frontend/components/container.cljs

```diff
@@ -583,8 +583,9 @@
                                      (remove string/blank?))]
                  (doseq [page pages]
                    (let [page (util/safe-page-name-sanity-lc page)
-                         [db-id block-type] (if (= page "contents")
-                                              ["contents" :contents]
+                         [db-id block-type] (case page
```

Comment:
This is not the correct way to add `:page-graph` support.

To add a new config item:

- add config key to config schema
- add config key and documentation to config template
- add optional default values to config.cjs
- add setter and getter fn
- use the setter & getter fn

BTW, The newly introduced setting `:page-graph` is counterintuitive.
The current setting is `:default-home {:page "Changelog", :sidebar "Contents"}`. The keys reflect UI areas.

Could you provide more information on why you want to add a `page-graph` setting?